CloudSQLReplicator/google/cloud/pontem/sql/replicator/util/kms.py

```diff
@@ -0,0 +1,169 @@
+# Copyright 2019 The Pontem Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""KMS utility Module."""
```

Comment:
i don't immediately know what KMS is, so probably need more of a comment

Reply:
Expanded KMS in module docstrings.